src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  height: 60px;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.nav {
+  text-align: center;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: black;
+  text-decoration: none;
+  padding: 0;
+  margin: 0;
+  text-transform: uppercase;
+  position: relative;
+}
+
+nav ul li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link.is-active {
+  color: var(--blue-color);
+  position: relative;
+  height: 60px;
+}
+
+.nav__link.is-active::after {
+  content: '';
+  display: inline-block;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  background-color: var(--blue-color);
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
 }
```

Comment:
This is a good use of the `:hover` pseudo-class to change the link color on hover.